src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/SEC101_106_AzureStorageAccountLegacyCredentials.cs

```diff
@@ -72,7 +72,7 @@ private static byte[] ConvertFromBase32(string inputString)
 
             if (string.IsNullOrEmpty(inputString))
             {
-                return Array.Empty<byte>();
+                return new byte[0];
```

Comment:
>new [](http://example.com/codeflow?start=23&length=3)

DRY out your code `don't repeat yourself`. declare a static readonly empty byte array in a logical place and utilize it from the two rule locations you have found.